test/mocks/statement.of.capital.mock.ts

```diff
@@ -3,11 +3,10 @@ import { StatementOfCapital } from "private-api-sdk-node/dist/services/confirmat
 export const mockStatementOfCapital: StatementOfCapital = {
   classOfShares: "Ordinary",
   currency: "GBP",
-  numberAllotted: 10,
-  aggregateNominalValue: 1,
+  numberAllotted: "10",
```

Comment:
spotted these were the wrong types, numbers instead of strings and totalCurrency doesn't exist on the StatementOfCapital so removed it.